ports/nearobject-framework/vcpkg.json

```diff
@@ -1,41 +1,42 @@
-{
-  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
-  "name": "nearobject-framework",
-  "version-string": "0.0.1",
-  "dependencies": [
-    "catch2",
-    "nlohmann-json",
-    "magic-enum",
-    "cli11",
-    "wil",
-    {
-      "name": "vcpkg-cmake",
-      "host": true
-    },
-    {
-      "name": "vcpkg-cmake-config",
-      "host": true
-    }
-  ],
-  "features": {
-    "msvc-link-static-runtime": {
-      "description": "Enables statically linking against the VC++ runtime, required for UMDF drivers",
-      "dependencies": [
-        {
-          "name": "nearobject-framework"
-        }
-      ],
-      "supports": "windows"
-    },
-    "windows-build": {
-      "description": "Configures the project for an official Windows build",
-      "dependencies": [
-        {
-          "name": "nearobject-framework"
-        }
-      ],
-      "supports": "windows"
-    }
-  },
-  "builtin-baseline": "779ce74ef67d3e12d904da1b15f9ed5626d5f677"
-}
+{
+  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
+  "name": "nearobject-framework",
+  "version-string": "0.0.2",
```

Comment:
Not sure why this whole file is being flagged as changed. The only changes are the version bump to 0.0.2 on this line, and the addition of "plog" on line 10.